modules/rosa-configure.adoc

```diff
@@ -195,12 +195,37 @@ Verify that the AWS quotas are configured correctly in a specific region:
 $ rosa verify quota --region=us-west-2
 ----
 
+[id="rosa-download-rosa-client_{context}"]
+== download rosa
+
+Download the latest compatible version of the Red Hat OpenShift Service on AWS CLI (`rosa`).
+
+After you download `rosa`, extract the contents of the archive and add it to your path. See link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/rosa_cli/rosa-get-started-cli[Setting up the rosa CLI] for more details.
```

Comment:
Unless OSD/ROSA does something different that I'm not aware of, we shouldn't be using hard links to the docs (where we wish we could use xrefs). We have other workarounds (like adding additional resources sections after the module inclusions). I'd recommend removing this line and adding the xref as an additional resource (or talk to someone else on the ROSA team for what they normally do)

Reply:
This is something I'm trying to resolve in another PR at the moment as well; will double back.

Reply:
Done! Thanks for the help @bmcelvee :D